docker-compose.yml

```diff
@@ -91,6 +94,7 @@ services:
       - zammad-railsserver
     volumes:
       - zammad-config-nginx:/etc/nginx/sites-enabled:ro
+      - zammad-storage:/opt/zammad/storage:ro
```

Comment:
I don't think nginx requires access on the storage/ directory. I believe it's being proxied by rails.

Reply:
You are right. I removed it again and added a comment for why zammad-var is required there.